website/querylog.py

```diff
@@ -22,11 +22,17 @@ def __init__(self, **kwargs):
             self.start_rusage = resource.getrusage(resource.RUSAGE_SELF)
         self.attributes = kwargs
         self.running_timers = set([])
-        self.set(
-            start_time=dtfmt(self.start_time),
-            pid=os.getpid(),
-            loadavg=os.getloadavg()[0],
```

Comment:
Your PR description says it's about `getrusage`, but it seems to be this is the line you're interested in, right? Is it that `os.getloadavg()` doesn't exist on Windows?

If that is the case, in order to reduce duplication somewhat, can you change it to the following instead:

```py
loadavg = os.getloavg()[0] if not IS_WINDOWS else None
self.set(
    # ...
    loadavg=loadavg
    )
```

Other than that... whoops, sorry. I forgot to check the docs apparently 😅

Reply:
Indeed. `getloadavg()` is the problem.

Reply:
Applied your suggestion. Should I squash?

Reply:
We'll squash when we merge.